test/unit/reducers/dataReducer.spec.js

```diff
@@ -88,7 +88,7 @@ describe('dataReducer', () => {
           `${collection}.${doc}.newData.field`,
           data[doc].newData.field,
         );
-        expect(result).to.not.have.nested.property(
```

Comment:
This test was in place for a reason - not removing a value which is removed from the object will cause stale data in certain situations